src/pymmcore_plus/_cli.py

```diff
@@ -148,7 +149,8 @@ def mmstudio() -> None:  # pragma: no cover
         print("[magenta]run `mmcore install` to install a version of Micro-Manager")
         raise typer.Exit(1)
     cmd = ["open", "-a", str(app)] if PLATFORM == "Darwin" else [str(app)]
-    raise typer.Exit(subprocess.run(cmd).returncode)
+    with contextlib.chdir(str(mm)):
```

Comment:
Is there a better way than this to keep mypy from complaining that `mm` is `str | None`?

Reply:
This is essentially the same check as `if not app` above, so you could make it `if not mm or not app` and then mypy should narrow the type to str

Reply:
I though I tried that but I guess I didn't because it works now. Thanks.